### ISSUE TITLE
Allow custom subject prefixes

### ISSUE THREAD

**Issue by synth** [pboling/sanitize_email#21]
It'd be nice to have my own variable prepended to the subject rather than being limited to environment and email recipients.

Bonus points, allow this as proc so the prefix is dynamically generated on a per mail basis

**Comment by MacksMind** [comment]
@synth You can override the environment in the config block to be anything you like. For example:
```ruby
config[:environment] = "[#{ENV.fetch('AWS_STACK_NAME', Rails.env)}]"
```
Didn't try a proc to see if it can be dynamic.

**Comment by pboling** [comment]
It didn't support `proc` but I'm adding that now.  Also, hi @MacksMind !

**Comment by pboling** [comment]
Released in 2.0.5
